### ISSUE TITLE
a11y smoketests still flaky

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#286855]
https://github.com/microsoft/vscode/actions/runs/20869537710/job/59968258114?pr=286850

With this error, 

```
 1) VSCode Smoke Tests (Electron)
       Accessibility
         Workbench
           workbench has no accessibility violations:
     Error: Accessibility violations found:

[aria-required-children] Certain ARIA roles must contain particular children (critical)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/aria-required-children?application=axeAPI
  Element: .list_id_5
    Class: monaco-list list_id_...
    HTML: <div class="monaco-list list_id_..." tabindex="0" role="list" aria-label="Error: Activating ex..." aria-multiselectable="true" data-keybinding-cont...="17">
    Issue: Fix any of the following:
  Element has children which are not allowed: [role=dialog]
```

disabling for now
